File: Usuario.java
class Usuario {
    
    // propiedades o atributos
    private String nombre;
    private int codigo;
    private String telefono;
    
    // constructor
    public Usuario() {
        this.nombre = "Desconocido";
        this.codigo = 0;
        this.telefono = "0";
    }
    
    // constructor
    public Usuario(String nom, int cod, String tel) {
        this.nombre = nom;
        this.codigo = cod;
        this.telefono = tel;
    }
    
    // metodos get
    public String getNombre() {
        return this.nombre;
    }
    
    public int getCodigo() {
        return this.codigo;
    }
    
    public String getTelefono() {
        return this.telefono;
    }
    
    // metodos set
    public void setNombre( String nom ) {
        this.nombre = nom;
    }
    
    public void setCodigo( int codigo ) {
        this.codigo = cod;
    }
    
    public void setTelefono( String tel ) {
        this.telefono = tel;
    }
    
}

File: Usuario.php
<?php
	//echo "HOLA MUNDO";
	class Usuario{
		// propiedades o atributos
		private $nombre;
		private $codigo;
		private $telefono;
		
		// constructor
		public __construct($nombre, $codigo, $telefono){
			$this->nombre = $nombre;
			$this->codigo = $codigo;
			$this->telefono = $telefono;
		}
		
		//metodos get
		public getNombre(){
			return $this->nombre;
		}
		
		public getCodigo(){
			return $this->codigo;
		}
		
		public getTelefono(){
			return $this->telefono;
		}
		
		// metodos set
		public setNombre( $nom ){
			$this->nombre = $nom;
		}
		
		public setCodigo( $cod ) {
			$this->codigo = $cod;
		}
		
		public setTelefono( $tel ) {
			$this->telefono = $tel;
		}
	}
?>